src/components/molecules/ForgotPasswordStage1.tsx

```diff
@@ -48,16 +42,14 @@ const ForgotPasswordStage1 = ({
       <Spacer />
       <Spacer />
       {watch('phoneNumber')?.match(/^0[0-9]{9}$/) ? (
-        <Button w="full" onPress={() => setStage(stages.second)}>
+        <Button w="full" onPress={handleSubmit(continueToNextStage)}>
           {t('auth.continue')}
         </Button>
```

Comment:
I believe Note uses another regex to validate the phone number here. Consider discussing which one to use?

This is the said regex `/^((((\+66|66|0)\d{2})-?\d{3}-?\d{4})|(-))$/`